MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
+        </td>
+    </tr>
+	</tr>
+</table>
+
+## SfCards
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfCardViewStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfCardViewTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfCardView without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfCardViewTheme">CommonTheme</x:String>
+                <Color x:Key="SfCardViewBorderColor">Red</Color>
+				 <Color x:Key="SfCardViewIndicatorColor">Green</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+        <tr>
+        <td>
+            SfCardViewBorderColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the border in the card view.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	  <tr>
+        <td>
+            SfCardViewIndicatorColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the indicator of the card view.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	       <tr>
+        <td>
+            SfCardViewNormalBackround  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of the card.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+</table>
+
+## SfRadialGauge
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="55">
+            SfRadialGaugeStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+            SfRadialGaugeTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfRadialGauge without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, you can customize the appearance of SfRadialGauge without merging common theme resources and control style resource dictionaries.